tests/testArgumentParser.py

```diff
@@ -289,13 +290,15 @@ def testAtFile(self):
     def testLogDest(self):
         """Test --logdest
         """
-        logFile = "tests_argumentParser_testLog_temp.txt"
-        self.ap.parse_args(
-            config = self.config,
-            args = [DataPath, "--logdest", logFile],
-        )
-        self.assertTrue(os.path.isfile(logFile))
-        os.remove(logFile)
+        logFile = tempfile.NamedTemporaryFile()
+        try:
+            self.ap.parse_args(
+                config = self.config,
+                args = [DataPath, "--logdest", logFile.name],
+            )
+            self.assertTrue(os.path.isfile(logFile.name))
```

Comment:
`parse_args` does return the logger object so in theory we could ask that object to free up the resource but I don't think there is the opposite of the `pex_logging` `addDestination` method.